plugin.json

```diff
@@ -27,85 +27,108 @@
         },
         {
             "key": "database",
-            "name": "Snowflake Database Name",
+            "name": "Snowflake database name",
             "type": "string",
             "required": true
         },
         {
             "key": "dbschema",
-            "name": "Snowflake Database Schema Name",
+            "name": "Snowflake database schema",
             "type": "string",
             "default": "PUBLIC",
             "required": true
         },
         {
             "key": "table",
-            "name": "Snowflake Table Name",
+            "name": "Snowflake table name",
             "type": "string",
             "required": true,
             "hint": "Will be created if doesn't already exist."
         },
         {
             "key": "warehouse",
-            "name": "Snowflake Virtual Warehouse Name",
+            "name": "Virtual Warehouse to use for copying files",
             "required": true,
-            "type": "string",
-            "hint": "Warehouse to be used for ingesting from stage."
+            "type": "string"
         },
         {
             "key": "role",
-            "name": "Snowflake Role",
+            "name": "Specify a role to access Snowflake with",
             "type": "string",
-            "hint": "Optional, overrides the default role of the user."
+            "hint": "Optional, overrides the default role for the user."
         },
         {
             "key": "stageToUse",
-            "name": "Object Storage",
+            "name": "Stage Type",
             "type": "choice",
-            "choices": ["S3"],
-            "hint": "Object storage platform that will be used for staging data for Snowflake export."
+            "required": true,
+            "choices": ["S3", "GCS"]
         },
         {
             "key": "stage",
-            "name": "Snowflake Stage Name",
+            "name": "Name to use for external stage",
             "type": "string",
             "required": true,
-            "hint": "External (object storage) stage where files will be stored before final export."
+            "hint": "Will be created if doesn't already exist."
         },
         {
-            "key": "eventsToIgnore",
-            "name": "Ignored Events",
+            "key": "bucketName",
+            "name": "Bucket Name",
```

Comment:
My only nitpick is that now config again has inconsistent capitalization. E.g. "S3 Bucket Region" but "Snowflake table name". Let's stick to Title Case OR Sentence case for all config entry names.